threema-connector/pom.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>threema.connector</groupId>
+  <artifactId>threema-connector</artifactId>
+  <version>10.0.0-SNAPSHOT</version>
+  <packaging>iar</packaging>
+ 
+ <!--
+ <dependencies>
+      <dependency>
+       <groupId>ch.threema.apitool</groupId>
+       <artifactId>msgapi-sdk-java</artifactId>
+       <version>2.0.0</version>
+     </dependency>
+ </dependencies>
+-->
```

Comment:
remove me

Reply:
add a maven build step however, that fetches the JAR from theema servers and adds it into your lib directory ... then you can also omit the "binary" adding from the git-history ... and your repo will stay fast and lightweight.

Reply:
here's a maven build plugin that should do what you need:
https://github.com/maven-download-plugin/maven-download-plugin#wget-goal

Reply:
Implemented via maven antrun plugin